qiskit_ibm_runtime/utils/json.py

```diff
@@ -270,9 +274,8 @@ def object_hook(self, obj: Any) -> Any:
                     obj_val, self.__read_parameter_expression, False
                 )
             if obj_type == "Instruction":
-                return _decode_and_deserialize(
-                    obj_val, qpy_serialization._read_instruction, False
-                )
+                circuit = _decode_and_deserialize(obj_val, qpy_serialization.load)[0]
+                return circuit.data[0][0]
```

Comment:
I think having a comment here on what this is doing would be useful. Something like:

```
# Standalone instructions are encoded as the sole instruction in a QPY serialized circuit
# to deserialize load qpy circuit and return first instruction object in that circuit.
```